Marcio/registrar revamp (#2052)
cloudflare/cloudflare-docs: products/registrar/src/content/index.md

pcx-content: landing-page
---

# Cloudflare Registrar

![Registrar Logo](./static/registrar-color-logo.png)

> Manage your domain with Cloudflare Registrar and add an additional layer of security to your DNS records for free. Cloudflare Registrar also offers redacted WHOIS information by default and will only charge you what we pay to the registry for your domain. No markup. No surprise fees.

Reviewer: ```suggestion
Manage your domain with Cloudflare Registrar and add an additional layer of security to your DNS records for free. Cloudflare Registrar also offers redacted WHOIS information by default and will only charge you what is paid to the registry for your domain. No markup. No surprise fees.
```
